examples/rbd/block-pod-rx.yaml

```diff
@@ -0,0 +1,30 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: pod-block-rx-volume
+  labels:
+    app: pod-block-rx-volume
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      app: pod-block-rx-volume
+  template:
+    metadata:
+      labels:
+        app: pod-block-rx-volume
+    spec:
+      containers:
+        - name: centos
+          image: quay.io/centos/centos:latest
+          imagePullPolicy: IfNotPresent
+          command: ["/bin/sleep", "infinity"]
+          volumeDevices:
+            - name: data
+              devicePath: /dev/xvda
+      volumes:
+        - name: data
+          persistentVolumeClaim:
+            claimName: rx-pvc
```

Comment:
where does `rx-pvc` come from?

Reply:
These templates can be used by both ROX and RWX PVC types, hence I just named it `rx-pvc`

These should be created by the test case. In the test case we are overriding this with:
```
app.Spec.Template.Spec.Volumes[0].PersistentVolumeClaim.ClaimName = pvc.Name
```

Reply:
Until now, all the examples can be applied without modifications. Please re-use one of the existing PVC-names for this example too.

Comments in the examples are not forbidden either, and it may make things easier to understand too!

Reply:
I saw they are not consistent in all the examples, just that I didn't want to create more yamls for PVC's for just this tests, hence decided to reuse existing pvc.yaml with some tweaking in the tests itself like:

```
pvc.Spec.AccessModes = []v1.PersistentVolumeAccessMode{v1.ReadWriteMany}
volumeMode := v1.PersistentVolumeBlock                                  
pvc.Spec.VolumeMode = &volumeMode                                       
```


```
$ git grep "claimName"  examples/rbd/
examples/rbd/block-pod-clone.yaml:        claimName: block-pvc-clone
examples/rbd/pod-clone.yaml:        claimName: rbd-pvc-clone
examples/rbd/pod-restore.yaml:        claimName: rbd-pvc-restore
examples/rbd/pod.yaml:        claimName: rbd-pvc
examples/rbd/raw-block-pod.yaml:        claimName: raw-block-pvc
```

Anyway, I can just go with what's inside pvc.yaml :+1: